### ISSUE TITLE
Record pipeline url on trigger

### ISSUE THREAD

**Issue by finestructure** [SwiftPackageIndex/SwiftPackageIndex-Server#983]
We're currently not making use of the response to the trigger POST request:

```swift
            return Gitlab.Builder.postTrigger(client: client,
                                              cloneURL: $0.package.url,
                                              platform: platform,
                                              reference: reference,
                                              swiftVersion: swiftVersion,
                                              versionID: versionId)
                .map { $0.status }
```

We're getting back the following:

```
JSON: ["committed_at": null, "id": 268063339, "detailed_status": ["icon": "status_pending", "text": "pending", "label": "pending", "favicon": "https://gitlab.com/assets/ci_favicons/favicon_status_pending-5bdf338420e5221ca24353b6bff1c9367189588750632e9a871b7af09ff6a2ae.png", "group": "pending", "illustration": null, "details_path": "/finestructure/swiftpackageindex-builder/-/pipelines/268063339", "has_details": false, "tooltip": "pending"], "created_at": "2021-03-10T07:32:26.656Z", "user": ["id": 501391, "state": "active", "name": "Sven A. Schmidt", "avatar_url": "https://assets.gitlab-static.net/uploads/-/system/user/avatar/501391/avatar.png", "web_url": "https://gitlab.com/sven.schmidt", "username": "sven.schmidt"], "sha": "321e40a0bf375dc262f4399548f332671d10fe32", "duration": null, "tag": false, "updated_at": "2021-03-10T07:32:26.782Z", "finished_at": null, "ref": "linux-build-fixes", "before_sha": "0000000000000000000000000000000000000000", "yaml_errors": null, "coverage": null, "web_url": "https://gitlab.com/finestructure/swiftpackageindex-builder/-/pipelines/268063339", "status": "pending", "started_at": null]
```

from which we can extract `web_url` and populate the `Build` stub:

```swift
            return Build.trigger(database: database,
                          client: client,
                          platform: pair.platform,
                          swiftVersion: pair.swiftVersion,
                          versionId: trigger.versionId)
                .flatMap { _ in
                    Build(versionId: trigger.versionId,
                          platform: pair.platform,
                          status: .pending,
                          swiftVersion: pair.swiftVersion)
                        .create(on: database)
                }
```

We do record `web_url` later on when the builds report back but it'd be nice to have it in there on trigger. This would allow us to inspect pending builds more easily.

**Comment by finestructure** [comment]
Yeah, it's not logging in CI:

<img width="488" alt="CleanShot 2021-03-10 at 12 05 01@2x" src="https://user-images.githubusercontent.com/65520/110620050-dc8a5f00-8198-11eb-8e18-d9fc9f5b9235.png">

https://gitlab.com/finestructure/swiftpackageindex-builder/-/jobs/1086379448

**Comment by finestructure** [comment]
This was implemented in #1277
